glad/generator/c/templates/base_template.h

```diff
@@ -66,6 +66,9 @@ extern "C" {
 {% else %}
 typedef struct Glad{{ feature_set.name|api }}Context {
     void* userptr;
+{% if options.loader %}
+    void* handle;
```

Comment:
This should be at the end of the struct and be named `glad_loader_handle` to make sure this doesn't 'accidentally' get used for anything else by a user.

Reply:
Fixed, and also added the `_glad` prefix to the statically generated variable for consistency. (Now that we can easily use a longer name due to the template macro)